src/java/org/apache/cassandra/db/Keyspace.java

```diff
@@ -337,7 +348,7 @@ private Keyspace(String keyspaceName, boolean loadSSTables)
         for (TableMetadata cfm : metadata.tablesAndViews())
         {
             logger.trace("Initializing {}.{}", getName(), cfm.name);
-            initCf(Schema.instance.getTableMetadataRef(cfm.id), loadSSTables);
```

Comment:
for me: this should no longer be null as the schema block means we see a consistent view of schema, so will see all ref and metadata protected.